library/src/main/kotlin/org/quiltmc/qkl/library/client/screen/QuiltScreen.kt

```diff
@@ -14,17 +14,16 @@
  * limitations under the License.
  */
 
-@file:Environment(EnvType.CLIENT)
+@file:ClientOnly
```

Comment:
This entire file gives warnings as extensions like `public val Screen.buttons get() = (this as QuiltScreen).buttons` are meaningless when QuiltScreen is already injected into Screen. Is this a left-over from before it was injected and should be removed?